torch/testing/_internal/common_nn.py

```diff
@@ -3692,7 +3693,7 @@ def fractional_max_pool3d_test(test_case):
         check_gradgrad=False,
         desc='gelu_activation',
         with_tf32=True,
-        tf32_precision=0.01,
+        tf32_precision=0.05,
```

Comment:
Isn't it worrying? Did the new cublas version come out?

Reply:
This is the cublas in CUDA 11.2.1 which we have never looked through the tests greenish.